python/cudf/cudf/core/series.py

```diff
@@ -5200,14 +5211,22 @@ def value_counts(
                 index=cudf.Index([], dtype=self.dtype),
             )
 
-        res = self.groupby(self, dropna=dropna).count(dropna=dropna)
+        res = self.groupby(self, dropna=dropna)
+
+        res = res.count(dropna=dropna)
+
```

Comment:
```suggestion
        res = self.groupby(self, dropna=dropna).count(dropna=dropna)
```

was this a necessary change?